Sources/RInAppMessaging/Utilities/BundleInfo.swift

```diff
@@ -53,10 +53,6 @@ internal extension Bundle {
         module
         #else
 
```

Comment:
here I could just simply return `assetsBundle(bundleIdPhrase: "RInAppMessagingResources")`. The difference is that the current implementation tries to get resources based on IAM SDK bundle path where the other solution iterates over all available resource bundles. The current approach, to me, seems more precise and safe in terms of name conflicts etc (?). But I'm not sure if that's a good approach